src/wazuh_mcp_server/utils/validation.py

```
@@ -195,13 +239,33 @@ def sanitize_string(input_str: str, max_length: int = 1000) -> str:
 
 def validate_json_payload(payload: Any, max_size: int = 10000) -> Dict[str, Any]:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)